grml-crypt

```diff
@@ -182,7 +181,7 @@ function actionStart
   if [ ! -b "$TARGET_" ]; then
     notice "Operating on a file"
     isExistent "$TARGET_" die "image does not exist"
-    TARGET_=$(findNextFreeLoop) || die "could not find a free loop device"
+    TARGET_=$(findNextFreeLoop "eprint" "") || die "could not find a free loop device"
```

Comment:
I don't understand this change :thinking:

Reply:
reverted to relying on the defaults